includes/class-wcpdf-install.php

```diff
@@ -347,6 +347,13 @@ protected function upgrade( $installed_version ) {
 			update_option( 'wpo_wcpdf_settings_debug', $debug_settings );
 		}
 
+		// 2.8.2: enable number store migration
+		if ( version_compare( $installed_version, '2.8.2', '<=' ) ) {
+			$debug_settings = get_option( 'wpo_wcpdf_settings_debug', array() );
+			$debug_settings['migrate_number_stores'] = 1;
+			update_option( 'wpo_wcpdf_settings_debug', $debug_settings );
+		}
```

Comment:
while doing my first tests, my tables were not being migrated and I was confused why until I realized that this was because my version was still at 2.8.2 and this flag was never set. While this is not a realistic scenario for the actual release, it did make me think about whether this is reliable enough (I'm thinking about the odd 'reinstall fonts' issue that comes along, even though that should be done automatically on each upgrade).

Would it be a big 'sacrifice' to do this the other way around? always migrate unless it's disabled? We could set the flag to disable migration in the install routine, which will only run for new installs.

Reply:
@Spreeuw but is `<=`

Reply:
yes but the `upgrade` method only runs when the version number _changes_
Like I say though, my own scenario with the same version number is not really relevant, I'm more worried about other scenarios where the upgrade procedure it's ran for whatever reason

Reply:
I believe this will have no effect because the settings callback runs first and will do the migration.